pkg_client/src/components/PKGVisualization.tsx

```diff
@@ -0,0 +1,83 @@
+// Natural language to PKG component
+
+import { Container } from "react-bootstrap";
+import { UserContext } from "../contexts/UserContext";
+import { useContext, useEffect, useState } from "react";
+import axios from "axios";
+import QueryForm from "./QueryForm";
+import Button from "react-bootstrap/Button";
+
+const PKGVisualization = () => {
+    const { user } = useContext(UserContext);
+    const [error, setError] = useState("");
+    const [image_path, setImagePath] = useState("");
+    const [query_info, setQueryInfo] = useState("");
+    const [result, setResult] = useState("");
+    const baseURL =
+        (window as any)["PKG_API_BASE_URL"] || "http://127.0.0.1:5000";
+
+    useEffect(() => {
+        getImage();
+    }, []);
+
+    const getImage = () => {
+        axios
+            .get(`${baseURL}/explore`, {
+                params: {
+                    owner_username: user?.username,
+                    owner_uri: user?.uri,
+                },
+                responseType: "blob",
+            })
+            .then((response) => {
+                setError("");
+                const imageURL = URL.createObjectURL(
+                    new Blob([response.data], {
+                        type: "image/png",
+                    })
+                );
+                setImagePath(imageURL);
+            })
+            .catch((error) => {
+                setError(error.message);
+                throw error;
+            });
+    };
+
+    const executeQuery = (query: string) => {
+        return axios
+            .post(`${baseURL}/explore`, {
+                sparql_query: query,
+                owner_username: user?.username,
+                owner_uri: user?.uri,
+            })
+            .then((response) => {
+                setError("");
+                console.log(response.data);
+                setQueryInfo(response.data.message);
+                setResult(response.data.result);
+            })
+            .catch((error) => {
+                setError(error.message);
+                throw error;
+            });
+    };
+
+    return (
+        <Container>
+            <div>
+                <b>Here you can execute your own SPARQL queries to manage your PKG.</b>
```

Comment:
Should be queries to select statements within the PKG? From my understanding only SELECT queries are accepted.